{"name": "test_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'", "trace": "self = <test_case.test_login.TestLogin object at 0x000001C4E770DDA0>\n\n    @allure.story(\"登录已存在用户\")\n    def test_login(self):\n        \"\"\"\n        登录已存在的用户\n        :return:\n        \"\"\"\n>       lp = LoginPage(self.driver)\nE       AttributeError: 'TestLogin' object has no attribute 'driver'\n\ntest_login.py:30: AttributeError"}, "description": "\n        登录已存在的用户\n        :return:\n        ", "attachments": [{"name": "失败截图20211026090554", "source": "d2cd2a4c-f744-4ce8-9d93-d7ac49224198-attachment.png", "type": "image/png"}], "start": 1635210352547, "stop": 1635210354636, "uuid": "6a839246-603d-40d4-9bba-dc2c72ef3d75", "historyId": "29097a56a45475cf7444439fb6057a04", "testCaseId": "da88586f2e77db1a02b8abcd6761bb9a", "fullName": "test_case.test_login.TestLogin#test_login", "labels": [{"name": "story", "value": "登录已存在用户"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-26V1U3BL"}, {"name": "thread", "value": "18420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}